src/features/implementationProvider.ts

```diff
@@ -19,6 +19,6 @@ export default class CSharpImplementationProvider extends AbstractSupport implem
             }
 
             return response.QuickFixes.map(fix => toLocation(fix));
-        });
+        }).catch();
```

Comment:
Can we write these all the same way?

Reply:
The function here is returning a ProviderResult and not a promise hence I am unable to put an await here, becuase that would require the return type to be a promise